### PR TITLE
fixed #214 removed height property from cards to make responsive

### LINKED ISSUE #214: Responsive Issue
**page:-** #https://finder.usmans.me/repos/javascript
 

- The tabs rendering in **finder.usmans.me** web page are having issue with responsiveness.
- Can I assigned to this issue.

Screenshot from 2024-10-01 16-46-27

### PR DESCRIPTION
**Pull request by love-sonkar**
fixed #214 
i have fixed the responsive issues in cards.
Now cards content is not overflowing.

image

### CODE REVIEW

**Comment by max-programming** [comment]
Thanks for submitting this fix! Would you be interested in addressing some other responsive issues I mentioned below?

hacktoberfest-projects-git-for-fbdb9a-max-programmings-projects vercel app_repos_javascript(iPhone 12 Pro)

File: package-lock.json
Comment (max-programming) [inline package-lock.json:1]: Can you delete the package lockfile please?
As this project uses `pnpm`

Comment (love-sonkar) [reply package-lock.json:1]: sure

**Comment by love-sonkar** [comment]
i have fixed the issue,
i'm sharing the image please review once
image


i'm not able to reproduce this issue other than that i have fixed all issue as you mentions.

Screenshot 2024-10-01 232931

**Comment by max-programming** [comment]
@love-sonkar The second image you shared is not a problem so ignore that

The sort and language buttons look fine but the header feels too cluttered

**Comment by love-sonkar** [comment]
we can do one thing, we can remove the contributers button from header when we go 768px

**Comment by love-sonkar** [comment]
i have given hidden property to the contributers button and added text wrap to sign in text.

image

**Comment by max-programming** [comment]
@love-sonkar How about having a hamburger menu instead for mobile?

We can use drawer component by DaisyUI
https://daisyui.com/components/drawer/

**Comment by love-sonkar** [comment]
Hey i would like to suggest somthing. so our website is all about finding repository for hacktoberfest and i think most of the users visit this site in laptop desktop, so i don't think it's good idea to add an drawer component because our our visitors are all from laptop and desktop.

**Comment by max-programming** [comment]
@love-sonkar Even though most visitors use laptop/desktop, there is still a large number of users on phones, and it will increase. 
Not forcing you but if you're not comfortable then maybe create a new issue so someone else will take this up

image

**Comment by love-sonkar** [comment]
okay i will create new Issue because i'm little bit busy so i can't implement this

**Comment by love-sonkar** [comment]
can you merge and close this PR

**Comment by max-programming** [comment]
As of now something on the homepage looks messed up
image

**Comment by max-programming** [comment]
@love-sonkar Any updates?

**Pull request closed without merging**